nifi-bootstrap/src/test/java/org/apache/nifi/bootstrap/configuration/StandardConfigurationProviderTest.java

```diff
@@ -117,6 +120,34 @@ void testGetManagementServerAddress(@TempDir final Path applicationHomeDirectory
         assertEquals(MANAGEMENT_SERVER_ADDRESS, address.toString());
     }
 
+    @Test
+    void testGetAdditionalArguments(@TempDir final Path applicationHomeDirectory) throws IOException {
+        final Path bootstrapConfiguration = setRequiredConfiguration(applicationHomeDirectory);
+        // Properties in random order and containing some java.arg and some NOT java.arg arguments.
+        List<String> propertyNames = new ArrayList<>(
+                Arrays.asList("java.arg9", "java.arg2", "java.arg.my2", "non.java.arg.2", "java.arg1", "java.arg.memory", "java.arg", "java.arg.my1", "non.java.arg.3", "random.nothing"));
+        // The expected returned list of java.arg properties sorted in ascending alphabetical order.
+        List<String> expectedArguments = new ArrayList<>(
+                Arrays.asList("java.arg", "java.arg.memory", "java.arg.my1", "java.arg.my2", "java.arg1", "java.arg2", "java.arg9"));
+
+        final Properties bootstrapProperties = new Properties();
+        for (String propertyName : propertyNames) {
+            bootstrapProperties.put(propertyName, propertyName + ".value");
```

Comment:
Recommend either removing the `.value` addition, or defining a variable and reusing it here and below to ensure the same suffix is applied.